### PR TITLE
Implement `Restricted` component

### PR DESCRIPTION
**Pull request by SilasBerger**
This component is essentially a stripped-down version of the `Solution`-component: It...
- ...does not use `<details>`
- ...shows nothing at all (i.e. an empty `<div>` if the user doesn't have permission to view the item or isn't logged in
- ...only shows the `<Loader>` if the user is logged in

Note: just like with the `<Solution>`-component, the contents of a `<Restricted>` are still sent to the client. The idea is to allow admins to hide an element on the website, if...
- ...a given user or user group is not supposed to see it (yet), but it's fine if they find it anyway. This could for instance be used for progress pacing, as as a means to prevent "spoiling" a surprise or effect, or as a way differentiate contents for individual students or students groups (such as half-classes).
- ...a given element only makes sense for logged-in users, such as restricted OneDrive links.

### CODE REVIEW

File: docs/Komponentengalerie/restricted-elements.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<BrowserWindow>
+<Restricted id="43ba2a38-9612-402f-9daf-91204ff47a71">
+Dieser Link wird nur bei entsprechender Berechtigung angezeigt:
+
+[🔗 https://test.com/nicht-oeffentlicher-link](https://test.com/nicht-oeffentlicher-link)
+</Restricted>
+</BrowserWindow>
+
+:::warning[Inhalt ist nicht geheim]
+Obwohl der zugriffsbeschränkte Inhalt bei fehlender Berechtigung nicht im Browser angezeigt wird, ist er dennoch im Seitenquelltext enthalten. Versierte Nutzer:innen können also trotzdem darauf zugreifen.
```
Comment (lebalz) [inline docs/Komponentengalerie/restricted-elements.md:29]: Haben wir source-maps aktiv? Andernfalls ist der Inhalt nur in den minified js files zu finden... das ist nochmals schwieriger zu finden... aber auch mit aktiven source-maps ist es nicht offensichtlich, wo der Inhalt zu finden ist...

**Review by lebalz — COMMENTED** [review]
LGTM 🥳
Greetings from 🇨🇴

File: docs/Komponentengalerie/restricted-elements.md (hunk 1 above)
Comment (SilasBerger) [reply docs/Komponentengalerie/restricted-elements.md]: Guter Punkt! Ich habe das in der Warnung noch etwas "abgeschwächt" und auf der Gallery-Page die Use Cases aus diesem PR ergänzt. Damit sollte relativ klar sein, für welche Elemente man diese Komponenten problemlos verwenden darf, und wo es vielleicht zu riskant wäre.

**Comment by SilasBerger** [comment]
> LGTM 🥳 Greetings from 🇨🇴

Great, thanks for the review and enjoy your vacation! 🥳

**Pull request merged**